src/jit/CMakeLists.txt

```diff
@@ -9,7 +9,9 @@ include_directories("../inc")
 
 if (CLR_CMAKE_TARGET_ARCH_AMD64 OR CLR_CMAKE_TARGET_ARCH_I386)
   add_definitions(-DFEATURE_SIMD)
-  add_definitions(-DFEATURE_AVX_SUPPORT)
+  if (!CLR_CMAKE_PLATFORM_UNIX)
```

Comment:
Also, could you please use "NOT" instead of "!"? We do it this way in all the CMakeFiles.txt and in fact, I have not found the "!" documented anywhere.